docs/advanced_usage/9_parallelism.rst

```diff
@@ -21,6 +21,39 @@ SMAC supports multiple workers natively via Dask. Just specify ``n_workers`` in
     When using multiple workers, SMAC is not reproducible anymore.
 
 
+..warning::
+
```

Comment:
Seems like two black spaces are missing (before and after warning).

Reply:
You are right, fixed it